scripts/download.pl

```diff
@@ -56,6 +58,29 @@ sub localmirrors {
 	return @mlist;
 }
 
+sub remotemirrors {
+	my $remote = shift;
+	my $remotes = shift;
+	my $mirrors = shift;
+	my $append = shift;
+
+	for my $object_name (sort keys %{$remotes}) {
+		next if ($object_name ne $remote);
+		for my $element (keys @{ $remotes->{$object_name} }){
```

Comment:
Stupid question about perl... But not following... we use a JSON. We decode it... Why we have to loop the array instead of checking if the key exist and access it directly?

Reply:
Because I would like to check whether the last char is a `/`.

Reply:
ok but what about the first for?

Reply:
You may be right that it can also be done more simply.